src/servers/src/influxdb.rs

```diff
@@ -88,7 +88,7 @@ impl TryFrom<InfluxdbRequest> for RowInsertRequests {
 
             // timestamp
             let precision = unwrap_or_default_precision(value.precision);
```

Comment:
it's not strictly related to the changes, but a side observation - 
does it make sense to move `precision` calculation out of the `for line in &lines {` WDYT?

Reply:
Good catch!